Quorum/apis/price_feeds/price_feed_utils.py

```diff
@@ -40,32 +60,36 @@ def __init__(self):
         self.cache_dir = Path(__file__).parent / "cache" / self.get_name().value
         self.cache_dir.mkdir(parents=True, exist_ok=True)
         self.session = requests.Session()
-        self.memory: dict[Chain, dict[str, PriceFeedData]] = {}
+        self.memory: dict[str, PriceFeedData] = {}
```

Comment:
I'm not sure if this change is very safe... What if my some unlucky chance you'll have the same address on 2 chains? I do see you divide the actual cache on the file system to chain -> address but not in code.

Reply:
Lol, its not going to happen dont worry